platform/android/CPackAndroidDeployQt.cmake.in

```diff
@@ -1,6 +1,6 @@
-if(DEFINED ENV{KEYNAME}
-   AND DEFINED ENV{KEYPASS}
-   AND DEFINED ENV{STOREPASS})
+if(NOT "ENV{KEYNAME}x" STREQUAL "x"
+   AND NOT "ENV{KEYPASS}x" STREQUAL "x"
+   AND NOT "ENV{STOREPASS}x" STREQUAL "x")
```

Comment:
Just managed to build via docker locally, adding a $ worked for me:
```suggestion
if(NOT "$ENV{KEYNAME}x" STREQUAL "x"
   AND NOT "$ENV{KEYPASS}x" STREQUAL "x"
   AND NOT "$ENV{STOREPASS}x" STREQUAL "x")
```

Reply:
> Just managed to build via docker locally, adding a $ worked for me:

A lot of implicit knowledge required for expansion in `if` https://cmake.org/cmake/help/latest/command/if.html

> There is no automatic evaluation for environment or cache [Variable References](https://cmake.org/cmake/help/latest/manual/cmake-language.7.html#variable-references). Their values must be referenced as $ENV{<name>} or $CACHE{<name>} wherever the above-documented condition syntax accepts <variable|string>.